.circleci/config.yml

```
@@ -76,12 +76,14 @@ jobs:
 
       - run:
           name: Get Python running and install dependencies
+          # Install mne-bids dev version: https://github.com/mne-tools/mne-connectivity/pull/237
           command: |
             pip install git+https://github.com/mne-tools/mne-python@main
             curl https://raw.githubusercontent.com/mne-tools/mne-python/main/tools/circleci_dependencies.sh -o circleci_dependencies.sh
             chmod +x circleci_dependencies.sh
             ./circleci_dependencies.sh
             pip install .[doc]
+            pip install git+https://github.com/mne-tools/mne-bids@main
```

Saves `pip` the hassle of installing an old version just to immediately uninstall it
```suggestion
            pip install .[doc] git+https://github.com/mne-tools/mne-bids@main
```

---

Ah cool, didn't realise it would recognise for do that!

---

Yes the `.[doc]` is a positional command-line argument! You can even do stuff like
```
pip install -e .[doc] mne-bids -r requirement.txt` etc. and it will parse correctly